services/src/test/scala/cromwell/services/metadata/impl/sns/AwsSnsMetadataServiceActorSpec.scala

```diff
@@ -0,0 +1,59 @@
+package cromwell.services.metadata.impl.sns
+
+import java.time.OffsetDateTime
+
+import akka.actor.{ActorInitializationException, ActorRef, Props}
+import akka.testkit.{EventFilter, TestProbe}
+import com.typesafe.config.{Config, ConfigFactory}
+import cromwell.core.WorkflowId
+import cromwell.services.ServicesSpec
+import cromwell.services.metadata.{MetadataEvent, MetadataKey, MetadataValue}
+
+
+class AwsSnsMetadataServiceActorSpec extends ServicesSpec("SnsMetadata") {
+  import AwsSnsMetadataServiceActorSpec._
+
+  val registryProbe: ActorRef = TestProbe().ref
+
+  "An AwsSnsMetadataActor with an empty serviceConfig" should {
+    "fail to build" in {
+      EventFilter[ActorInitializationException](occurrences = 1) intercept {
+        system.actorOf(Props(new AwsSnsMetadataServiceActor(emptyConfig, emptyConfig, registryProbe)))
+      }
+    }
+  }
+
+  "An AwsSnsMetadataActor with a topic and configuration" should {
+    "successfully build" in {
+      system.actorOf(Props(new AwsSnsMetadataServiceActor(configWithTopic, emptyConfig, registryProbe)))
+    }
+
+    "process an event" in {
+      val actor = system.actorOf(Props(new AwsSnsMetadataServiceActor(configWithTopic, emptyConfig, registryProbe)))
+      actor ! event
```

Comment:
This isn't actually testing anything, other than that akka can send the message. You might prefer `?` which would return a future waiting for the actor to return something, or you might also want something more like `var actor = TestActorRef(new AwsSnsMetadataServiceActor(...))` which lets you introspect what's going on inside the actor after you've sent a message to it.